### PR TITLE
adding beta launch documentation

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As discussed, this is the addition of the beta launch document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # Notice
-**This repository is not yet active.**
+This repo is currently being beta tested. To find out how to contribute to the beta and give feedback, see our [Yari beta launch](BETA.md) document.
```
Comment (escattone) [inline README.md:2]: Should we make this bold?
```suggestion
**This repo is currently being beta tested. To find out how to contribute to the beta and give feedback, see our [Yari beta launch](BETA.md) document.**
```

File: BETA.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Basic contributions
+
+You'll find most of the information you need to edit Yari in this very repo's main [README file](README.md). Go there, have a read of the [setup instructions](README.md#setup), fork the repo, and try making some changes.
+
+If you want to find some legitimate changes to make while testing the beta, have a look at our [user-submitted documentation issues](https://github.com/mdn/sprints/issues), or think about a page you know of that could do with a better explanation or example!
+
+## Tooling
+
+To begin with, we are keeping the Yari tooling basic, with the intention of improving what's on offer as we get more feedback. As you'll find out from reading through the README, there are already a few fundamental CLI commands available, such as `start` (to create a live preview of what your document will look like when rendered on MDN), `create` (to add a new page), `move` (to move an existing page), etc.
```
Comment (escattone) [inline BETA.md:23]: Do you think we should be more explicit with the commands? Up to you.
```suggestion
To begin with, we are keeping the Yari tooling basic, with the intention of improving what's on offer as we get more feedback. As you'll find out from reading through the README, there are already a few fundamental CLI commands available, such as `yarn start` (to create a live preview of what your document will look like when rendered on MDN), `yarn content create` (to add a new page), `yarn content move` (to move an existing page), etc.
```

**Review by escattone — APPROVED** [review]
Just a few nits that I'll leave up to you to ignore or accept. Thanks @chrisdavidmills!

**Comment by chrisdavidmills** [comment]
Thanks @escattone ! I've accepted your changes and merged it.

**Pull request merged**